lib/serialize.js

```diff
@@ -124,13 +124,13 @@ export async function exportKeyPair({
 
 // imports key pair
 export async function importKeyPair({
-  id, controller, secretKeyMultibase, publicKeyMultibase
+  id, controller, secretKeyMultibase, publicKeyMultibase, type
 }) {
   if(!publicKeyMultibase) {
     throw new TypeError('The "publicKeyMultibase" property is required.');
   }
 
-  const keyPair = {id, controller};
+  const keyPair = {id, controller, type};
```

Comment:
The key pair type is always `Multikey` so we shouldn't allow this to be passed in, but rather we should forcibly set it.

Reply:
Updated [a69df4c](https://github.com/digitalbazaar/ecdsa-multikey/pull/16/commits/a69df4ce7b4b82bd967f57d4820df546cdb00d13)

Reply:
Should we also set the `type` property on the `keyPair` in `generate()` https://github.com/digitalbazaar/ecdsa-multikey/blob/main/lib/index.js#L25-L27?

Reply:
So not in `generate` but it can be done in:

https://github.com/digitalbazaar/ecdsa-multikey/blob/main/lib/index.js#L76

And, yes, same can be done over in ed25519-multikey.

Reply:
Hmm, wait, let's discuss elsewhere.